src/connectors/shmem/Makefile.am

```diff
@@ -23,5 +23,4 @@ shmem_la_LDFLAGS = -module $(san_ld_zdef_flag) \
 shmem_la_LIBADD = \
 	$(top_builddir)/src/common/libflux-internal.la \
 	$(top_builddir)/src/common/libflux-core.la \
-	$(ZMQ_LIBS) \
```

Comment:
shmem connector uses the `zsock` class, so this is still required.  Also it looks like `$(ZMQ_CFLAGS)` is missing so that should be added.